### PR TITLE
Update to AKS 1.9.9, RBAC enabled

### PR DESCRIPTION
**Pull request by dtzar**
## Purpose
- Updates to AKS 1.10.5, enables RBAC
- Updates relevant helm charts for RBAC
- Update az version 31 --> 41

### CODE REVIEW

File: provision-proctor/Readme.md [hunk 1]
```diff
@@ -14,5 +14,5 @@ az login
 #### Deploy proctor environment
 
 ```shell
-nohup setup.sh -i <subscriptionId> -l <resourceGroupLocation> -m <proctorName> -u <proctorNumber> -n <teamName> -e <totalTeams> > <proctorName><proctorNumber>.out &"
+nohup ./setup.sh -i <subscriptionId> -l <resourceGroupLocation> -m <proctorName> -u <proctorNumber> -n <teamName> -e <totalTeams> > <proctorName><proctorNumber>.out"
```
Comment (dcaro) [inline provision-proctor/Readme.md]: The command should run in the background to be resilient

File: provision-team/provision_aks.sh [hunk 2]
```diff
@@ -75,7 +75,7 @@ fi
 echo "Creating AKS Cluster..."
 (
     set -x
-    az aks create -g $resourceGroupName -n $clusterName -l $resourceGroupLocation --node-count 3 --generate-ssh-keys -k 1.9.6
+    az aks create -g $resourceGroupName -n $clusterName -l $resourceGroupLocation --enable-rbac --node-count 2 --generate-ssh-keys -k 1.10.3
```
Comment (dcaro) [inline provision-team/provision_aks.sh]: Please keep the cluster size to 3.

**Comment by dcaro** [comment]
Let's talk before adding RBAC in this deployment.

**Comment by rguthriemsft** [comment]
We need to hold off on moving to 1.10.3 until after dry run.  There were some issues last week with AZ Cli and user also did not seem to come up properly when I tested it.

**Comment by dcaro** [comment]
Approved as per our conversation.

**Pull request merged**